From b40a8334d0a81d88be7371fa2124ce30994d4f94 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Fri, 8 Jan 2016 14:27:36 +0200
Subject: Disable PERL_TRACK_MEMPOOL for debugging builds

This is a workaround for an ABI incompatibility between
-DDEBUGGING and normal builds.

Bug-Debian: https://bugs.debian.org/810326
Bug: https://rt.perl.org/Public/Bug/Display.html?id=127212
Patch-Name: debian/debugperl-compat-fix.diff
---
 perl.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/perl.h b/perl.h
index 1325de9b9..8f19b2895 100644
--- a/perl.h
+++ b/perl.h
@@ -176,7 +176,9 @@
 #  define pTHX_8	9
 #  define pTHX_9	10
 #  define pTHX_12	13
-#  if defined(DEBUGGING) && !defined(PERL_TRACK_MEMPOOL)
+/* PERL_TRACK_MEMPOOL temporarily disabled for DEBUGGING */
+/* see https://bugs.debian.org/810326 */
+#  if 0 && defined(DEBUGGING) && !defined(PERL_TRACK_MEMPOOL)
 #    define PERL_TRACK_MEMPOOL
 #  endif
 #else
